### DISCUSSION TITLE
Positively stumped trying an individual CoPilot subscription.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#51043] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

This is the page I'm directed to after choosing between either business or individual. goes nowhere.

https://github.com/settings/copilot

The 'save' object is dead

**Comment by iamcjwilliams** [comment]
@jpeek3456 Have you visited here: https://github.com/features/copilot

**Comment by ghost** [comment]
iamcjwilliams,

It's the same scenario.

https://i.imgur.com/jtMXt3q.png

Should I have a separate utility running before I join?

Additionally, The order of language implies that I cannot leave the box
unchecked if there's no page turn after pressing the dead object.

The start object is dead.

What am I doing wrong?

Where's the beef after all the AI hype?

This sign up process.... ridiculous.

@jpeek345

**Reply by iamcjwilliams** [reply]
Have you tried checking the box and moving forward?

**Comment by ghost** [comment]
iamcjwilliams,

While I appreciate your attention to my post I don't appreciate github
making an overly simplified page presenting a checkbox as if it were an
option when it is not.

Additionally an asterisk has always meant that it's dependency is legally
not far from the asterisk.

Github aka Microsoft has no such asterisk on this page for the public.

If so then please direct me to said asterisk.


Ty.

**Comment by iamcjwilliams** [comment]
@jpeek3456 In your image you have "allow" selected, not "block", have you tried changing to block, keeping the asterisk unchecked and moving forward?

**Reply by ghost** [reply]
Why does it matter?
Come to think of it, I experimented with the 'block' option originally _before_ I got to this point....proving that I'm going to see if github even lets me into their Co Pilot X club.
This isn't where I belong anyway.
